compiler/quilt/test/test_install.py

```diff
@@ -296,6 +335,18 @@ def test_quilt_yml_unknown_version(self):
         with assertRaisesRegex(self, command.CommandException, "Version 99.99 does not exist"):
             command.install("packages:\n- akarve/sales:v:99.99")
 
+    def test_quilt_yml_unknown_team(self):
+        table_data1, table_hash1 = self.make_table_data('table1')
+        contents1, contents_hash1 = self.make_contents(table1=table_hash1)
+        self._mock_tag('baz/bat', 'latest', contents_hash1, team='unknown')
+        self._mock_package('baz/bat', contents_hash1, '', contents1,
```

Comment:
Actually, you don't get a 404 if the team doesn't exist. Different teams are hosted on physically different servers; e.g., "unknown" would use the URL "https://unknown-registry.team.quiltdata.com". In an ideal world, this would result in a DNS error. In our case, it actually points to the wrong server and results in an SSL error.

So I'd suggest either simulating what's actually happening now - make the tag request raise an `SSLError` (I think `responses` library, used by `_mock_tag`, allows raising exceptions.), or just skip this test for now, until we fix the servers.

Reply:
Got it, thanks!